system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -116,6 +116,10 @@ parameter_types! {
 			version: [4, 0, 0, 0], // 0x04000000
 			epoch: 269568,
 		},
+		electra: Fork {
+			version: [5, 0, 0, 0], // 0x05000000
+			epoch: 362496,
```

Comment:
```suggestion
			epoch: 364032,
```

https://github.com/ethereum/pm/blob/master/Pectra/pectra-mainnet-plan.md#upgrade-information